README.md

```diff
@@ -1,25 +1,124 @@
-# AdminToolbox [![GitHub release](https://badgen.net/github/release/Rnen/AdminToolbox)](https://GitHub.com/Rnen/AdminToolbox/releases/) [![Github all releases](https://img.shields.io/github/downloads/Rnen/AdminToolbox/total.svg)](https://GitHub.com/Rnen/AdminToolbox/releases/) [![GitHub latest commit](https://badgen.net/github/last-commit/Rnen/AdminToolbox)](https://GitHub.com/Rnen/AdminToolbox/commit/) [![GitHub watchers](https://img.shields.io/github/watchers/Rnen/AdminToolbox.svg?style=social&label=Watch&maxAge=2592000)](https://GitHub.com/Rnen/AdminToolbox/watchers/)
-### This is a server plugin for the game **SCP:Secret Laboratory**. 
-**It offers multiple admin tools like** 
-* Friendly fire logs in server console & logs to file
-* Advanced Godmode, ability to turn off damage for players, instantkill
-  - "Advanced" in the sense of: 106 with ATGOD cannot be contained, amongst other things like this
-* Warp Points
-* Modifying player cordinates/position
-* Prevent the round from ending
-* Per-player stats (saved to file)
-* Breaking doors interacted with
-* Player Joined messages in server window
-
-and more commands, and other features that do not exist in the base game (yet).
-This plugin is also beeing constantly updated with more features, so stay tuned!
-
-If you want to follow the development, take a look at the [AdminToolbox Progress Board!](https://github.com/Rnen/AdminToolbox/projects/1)
-***
-## Plugin Installation & Usage:
-[Visit the Wiki Pages to learn more!](https://github.com/Rnen/AdminToolbox/wiki)
-  
-
-### Place any suggestions/problems in [issues](https://github.com/Rnen/AdminToolbox/issues)!
-
-# Thanks & Enjoy.
+<br>
+
+<div align = center>
+
+[![Badge Release]][Releases]   
+[![Badge Downloads]][Releases]   
+[![Badge Commit]][Commit]   
+[![Badge Watchers]][Watchers]
+
+<br>
+
+# Admin Toolbox
+
+*A **SMod** server plugin for **[SCP: Secret Laboratory]**.*
+
+---
+
+[![Button Installation]][Wiki]   
+[![Button Issues & Ideas]][Issues]   
+[![Button Development]][Projects]
+
+---
+
+</div>
+
+<br>
+<br>
+
+## Features
+
+-   Break, lock and unlock <br>
+    doors upon interaction
+    
+-   Teleporting Players
+
+-   Server Statistics
+
+-   Endless Rounds
+
+-   Jailing Players
+
+-   Infinite Items
+
+-   Warp Points
+
+-   Ghost Mode
+
+-   Instantkill
+
+
+<br>
+
+### Friendly Fire
+
+*For every act of friendly fire, the server* <br>
+*console and the log files will take note.* <br>
+
+*The Plugin also supports disabling certain* <br>
+*types of damage dealt between players.*
+
+<br>
+
+### Advanced GodMode
+
+Turn off damage for players.
+
+*'Advanced' in the sense of `106` with ATGOD* <br>
+*cannot be contained, amongst other things.*
+
+<br>
+
+### Player Statistics
+
+*Keeps track of player statistics and* <br>
+*stores them in a dedicated files.*
+
+#### Tracked
+
+- `Team Kills`
+- `Playtime`
+- `Deaths`
+- `Kills`
```

Comment:
Just a small change here, to reflect the order the stats are saved in, as well as add a "BanCount" stat i forgot.
```suggestion
- `Kills`
- `Team Kills`
- `Deaths`
- `Playtime` (in minutes)
- `Times Banned`
```